src/roaring64.c

```diff
@@ -803,6 +803,69 @@ bool roaring64_bitmap_run_optimize(roaring64_bitmap_t *r) {
     return has_run_container;
 }
 
+typedef struct min_max_sum64_s {
+    uint64_t min;
+    uint64_t max;
+    uint64_t sum;
+} min_max_sum64_s;
+
+static bool min_max_sum64_fnc(uint64_t value, void *param) {
+    min_max_sum64_s *mms = (min_max_sum64_s *)param;
+    if (value > mms->max) mms->max = value;
+    if (value < mms->min) mms->min = value;
+    mms->sum += value;
+    return true;  // we always process all data points
+}
+
+/**
+ *  (For advanced users.)
+ * Collect statistics about the bitmap
+ */
+void roaring64_bitmap_statistics(const roaring64_bitmap_t *r,
```

Comment:
This is currently prohibitively expensive, it seems. This function should be much more efficient.